e2e/windows/e2e-starter.sh

```diff
@@ -107,9 +112,11 @@ getMSIResourceID
 addJsonToFile "mcRGName" $MC_RESOURCE_GROUP_NAME
 addJsonToFile "clusterID" $CLUSTER_ID
 addJsonToFile "subID" $SUBSCRIPTION_ID
+addJsonToFile "orchestratorVersion" $KUBERNETES_VERSION
```

Comment:
The `orchestratorVersion` in `property-windows-template.json` is the version for the ARM template. I do have seen some transient windows vmss deployment errors using `v1.23.12` with cse package v1.25.6+. 

I think we can use $KUBERNETES_VERSION (or the windows package version more precisely) for the windows vmss deployment to reduce the chance of deployment failure. @junjiezhang1997 ?